.github/ISSUE_TEMPLATE/BUG-REPORT.yml

```diff
@@ -0,0 +1,69 @@
+name: 'Bug report'
+description: 'Report a bug to help us fix it.'
+labels: [ 'bug' ]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this form! We will respond as fast as we can. Don't forget to check
```

Comment:
Ik zou dit omdraaien, bijv: 
```Please check whether your issue might be a duplicate of an existing one before submitting. If your issue is not yet reported, thank you for filling out this form! We will respond as fast as we can.

Reply:
Ergens is dat logisch maar ik vind het wel lekker vriendelijk om positief te beginnen :P